backend/composer/api/filtersets.py

```diff
@@ -203,11 +204,27 @@ class NoteFilter(django_filters.FilterSet):
         field_name="connectivity_statement_id",
         queryset=ConnectivityStatement.objects.all(),
     )
+    include_system_notes = django_filters.BooleanFilter(
+        field_name="include_system_notes", method="get_notes", label="Include System Notes"
+    )
 
     class Meta:
         model = Note
         fields = []
 
+    @staticmethod
+    def get_notes(queryset, name, value):
+        if value:
+            return queryset
+        system_user = User.objects.get(username="system")
+        # BELOW LOGIC: if include_system_notes is False, show - system notes associated with "invalid" or "exported" + non_system_notes
```

Comment:
nitpik: "Below Logic" is Implied, there's no need to include that expression in the comment